cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/servlet/KafkaCruiseControlServletUtils.java

```diff
@@ -42,7 +44,7 @@ static String getClientIpAddress(HttpServletRequest request) {
     return request.getRemoteAddr();
   }
 
-  static String getProposalSummary(GoalOptimizer.OptimizerResult result) {
+  static String getProposalSummaryInString(GoalOptimizer.OptimizerResult result) {
```

Comment:
Can we avoid the name change in this case -- i.e. the return type is already specified as part of the function signature.
`getProposalSummaryInString` -> `getProposalSummary`